Documentation/building/windows-instructions.md

```diff
@@ -19,8 +19,13 @@ We also require that [Visual Studio 2015 Update 1](https://www.visualstudio.com/
 ## Building From the Command Line
 
 Open a [Visual Studio Command Prompt](http://msdn.microsoft.com/en-us/library/ms229859(v=vs.110).aspx).
-From the root of the repository, type `build.cmd`. This will build everything and run
-the core tests for the project. 
+
+`build.cmd` - Will cause basic tool initialization and build the default configuration for refs, libs, and packages.
+`build-tests.cmd` - Will build and run tests for the default configuration.
```

Comment:
Nit: For some reason the md file did not put the build-tests piece in a new line, which looks weird.